docs/framework/windows-workflow-foundation/using-workflowinvoker-and-workflowapplication.md

```diff
@@ -1,5 +1,6 @@
 ---
 title: "Using WorkflowInvoker and WorkflowApplication"
+description: This article describes workflow hosting with WorkflowInvoker and WorkflowApplication, two of the methods of hosting that Windows Workflow Foundation provides.
```

Comment:
```suggestion
description: This article describes workflow hosting using WorkflowInvoker and WorkflowApplication.
```

Reply:
@gewarren - How about this:

This article describes workflow hosting using WorkflowInvoker and WorkflowApplication in Windows Workflow Foundation.

Reply:
Sounds good.